tests/services/inventory/test_data/mock_gcp_results.py

```diff
@@ -274,11 +274,36 @@
         },
         "id": "project3:dataset1",
         "kind": "bigquery#dataset"
+    }],
+    # CAI uses project ID instead of number.
+    "project3": [{
```

Comment:
This is the raw API representation, so it needs to remain using the project number. This whole entry should be removed, it is not needed for the tests.

Reply:
Updated.